ignition-server/cmd/start.go

```diff
@@ -143,12 +143,15 @@ func setUpPayloadStoreReconciler(ctx context.Context, registryOverrides map[stri
 		Client:       mgr.GetClient(),
 		PayloadStore: payloadStore,
 		IgnitionProvider: &controllers.LocalIgnitionProvider{
-			ReleaseProvider: &releaseinfo.RegistryMirrorProviderDecorator{
-				Delegate: &releaseinfo.CachedProvider{
-					Inner: &releaseinfo.RegistryClientProvider{},
-					Cache: map[string]*releaseinfo.ReleaseImage{},
+			ReleaseProvider: &releaseinfo.ProviderWithOpenShiftImageRegistryOverridesDecorator{
+				Delegate: &releaseinfo.RegistryMirrorProviderDecorator{
+					Delegate: &releaseinfo.CachedProvider{
+						Inner: &releaseinfo.RegistryClientProvider{},
+						Cache: map[string]*releaseinfo.ReleaseImage{},
+					},
+					RegistryOverrides: registryOverrides,
 				},
-				RegistryOverrides: registryOverrides,
+				OpenShiftImageRegistryOverrides: util.ConvertImageRegistryOverrideStringToMap(os.Getenv("OPENSHIFT_IMG_OVERRIDES")),
```

Comment:
this looks breaking as well from previous GA behavior: but maybe I am wrong

Reply:
This is not a breaking change. If the env var is not present, then nothing different should happen